modules/solid_mechanics/test/tests/power_law_creep/tests

```diff
@@ -127,4 +127,49 @@
       detail = 'calculate a perfect Jacobian with the small strain formulation'
     []
   []
+  [composite_power_law_creep]
+    type = 'CSVDiff'
+    input = 'composite_power_law_creep.i'
+    csvdiff = 'composite_power_law_creep_tests_soln_0011.csv'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    requirement = "The system shall provide a phase-dependent power law creep"
+    detail = 'that works with different material properties.'
+    issues = '#28368'
+  []
+  [composite_power_law_creep_single_material]
+    type = 'CSVDiff'
+    input = 'composite_power_law_creep_single_material.i'
+    csvdiff = 'composite_power_law_creep_single_material_tests_soln_0011.csv'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    requirement = "The system shall provide a phase-dependent power law creep"
+    detail = 'that gives the same output as power_law_creep.i if given the same materials to all the phases.'
+    issues = '#28368'
+  []
+  [except_composite_power_law_activation_energy]
+    type = 'RunException'
+    input = 'composite_power_law_creep_activation_energy_error_test.i'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    expect_err = 'activation energy must be equal to the number of swithing functions'
+    requirement = "The system shall produce an error"
+                  " if the input activation energies are not the same length as the switching functions provided."
+    issues = '#28368'
+  []
+  [except_composite_power_law_coefficient]
+    type = 'RunException'
+    input = 'composite_power_law_creep_dorn_constant_error_test.i'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    expect_err = 'number of Dorn constant must be equal to the number of switching functions'
+    requirement = "The system shall produce an error"
+                  " if the input coefficients are not the same length as the switching functions provided."
+    issues = '#28368'
+  []
+  [except_composite_power_law_n_exponent]
```

Comment:
Instead of creating an entirely new input file, try to use the cli_args option with a the input file for the composite_power_law_creep.i file.